core/src/main/java/com/hcl/ecomm/core/config/LoginServiceConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.hcl.ecomm.core.config;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "Login Service", description = "Component Usage Report Configurations")
```

Comment:
@nitheshkumar-hcl : config name should be Magento Login Service

Reply:
config name changed to Magento Login Service